b2/console_tool.py

```diff
@@ -429,9 +429,12 @@ class GetBucket(Command):
 
         If --showSize is specified, then display the number of files
         (fileCount) in the bucket and the aggregate size of all files
-        (totalSize). Note that this entails additional API calls, so
-        will incur additional latency, computation, and Class C
-        transactions.
+        (totalSize). Hidden files are accounted for in the number of
+        files, but not in the aggregate size. Each version of a file
```

Comment:
I believe that hidden files are included in the total size.  It's the hide markers that have size 0 and are not included in the total size.

Reply:
My experimentation with service indicates that hidden files are not included in the total size, so the comment is accurate. The values produced by `--showSIze` also consistently agreed with the values reported by the website, in my experience. I can't comment on what it "should" do without a deeper understanding of the architects' intent, only what I observed it to actually do. I didn't test extensively, but I did smoke test ~6-7 data points, and they always agreed between the two mechanisms.

Reply:
I should observe that I didn't try experiments with nearly as many files using my own real bucket and the real service. Just a couple files, which I would hide/unhide as necessary.

Reply:
The totals on the web site com from the usage/billing system, which can lag by several minutes.

Reply:
Okay, I'll retest and update the documentation accordingly.

Reply:
I uploaded some test files.  You can see them all with the new `--recursive` flag on `ls`:

```
$ python -m b2 ls --versions --recursive --long bwb-test2
4_z4899f00dac54390c6e2f0519_f10552ec61b4a3769_d20180319_m193043_c000_v0001059_t0056    hide  2018-03-19  19:30:43          0  a
4_z4899f00dac54390c6e2f0519_f101aa4b02532b543_d20180316_m211136_c000_v0001058_t0053  upload  2018-03-16  21:11:36          1  a
4_z4899f00dac54390c6e2f0519_f112317c7cf6f77fa_d20180316_m211105_c000_v0001054_t0007  upload  2018-03-16  21:11:05          4  a
4_z4899f00dac54390c6e2f0519_f1162d927fa37bf30_d20180316_m211112_c000_v0001055_t0044  upload  2018-03-16  21:11:12          2  b
4_z4899f00dac54390c6e2f0519_f114f1f9f8e436775_d20180316_m211105_c000_v0001060_t0022  upload  2018-03-16  21:11:05          8  c/c1
4_z4899f00dac54390c6e2f0519_f105eb80bde615bc3_d20180316_m211112_c000_v0001060_t0018  upload  2018-03-16  21:11:12         16  c/c2
4_z4899f00dac54390c6e2f0519_f108cfa8045e6ebf4_d20180317_m004218_c000_v0001058_t0056  upload  2018-03-17  00:42:18          4  d
```

And the answer from your code is correct:

```
$ python -m b2 get-bucket --showSize bwb-test2
{
    "accountId": "890dc49cef59",
    "bucketId": "4899f00dac54390c6e2f0519",
    "bucketInfo": {},
    "bucketName": "bwb-test2",
    "bucketType": "allPrivate",
    "corsRules": [],
    "fileCount": 7,
    "lifecycleRules": [],
    "revision": 1,
    "totalSize": 35
}
```

Reply:
Okay, so no changes are required to either my code or my documentation? If I'm understanding that correctly, do you still need me to change anything, or do you want to close your change request?